### DISCUSSION TITLE
BUG Report for Transformers 4.21.0: Can't load XLMRoberta checkpoints

### DISCUSSION THREAD

**Discussion by kkie02** [community/community#27631] [Other Feature Feedback, Questions, & Ideas] [answered]
Describe:
I used XLMRobertaForMaskedLM.from_pretrained('xlm-roberta-base') to continue pretraining, the training process was too long, so I save checkpoints regularly. I did this in Google Colab.
Several days ago, I can't load any saved checkpoints by using "cont_pre_trainer.train(resume_from_checkpoint=True)", there is always such an error:
RuntimeError: Error(s) in loading state_dict for XLMRobertaForMaskedLM:
	Missing key(s) in state_dict: "lm_head.decoder.weight", "lm_head.decoder.bias". 

The reason:
XLMRobertaForMaskedLM doesn't have "lm_head.decoder.weight", "lm_head.decoder.bias". And state_dict of a PyTorch module is an OrderedDict and it complains about missing keys. Maybe you should use such a command somewhere: load_state_dict(state_dict, strict=False)

How to solve it by myself:
I rollbacked transformers to version 4.20.1 and it worked then. 

Problem Conclusion:
**Transformers version 4.21.0 can't load checkpoints** that trained on both version 4.20.1 and version 4.21.0.
(Transformers version 4.20.1 works normally, I use it to process checkpoints trained on version 4.20.1 or version 4.21.0)

@LysandreJik @sgugger

**Comment by airtower-luna** [comment, marked as answer]
That sounds like you're trying to report a bug in a software which is hosted on GitHub. Maybe https://github.com/huggingface/transformers? In that case you should create an issue *in that repository*. I hope this helps! :slightly_smiling_face:
